plugins/emmc/fu-emmc-device.c

```diff
@@ -164,6 +165,17 @@ fu_emmc_device_probe (FuDevice *device, GError **error)
 		return FALSE;
 	}
 
+	/* ignore *rpmb and *boot* mmc block devices */
+	dev_regex = g_regex_new("mmcblk\\d$", 0, 0, NULL);
+	if (!g_regex_match(dev_regex, g_udev_device_get_name (udev_device), 0, NULL)) {
```

Comment:
Space before the bracket please. Also do we need to check for name!=Null?

Reply:
Fixed.
Checking for null is not strictly necessary  (https://fossies.org/linux/glib/glib/gregex.c line 1923), but better to have it, yes.